rollup.config.js

```diff
@@ -84,14 +84,21 @@ const plugins = [
   typescript({
     typescript: require('typescript'),
     cacheRoot: './node_modules/.cache/.rts2_cache',
+    useTsconfigDeclarationDir: true,
     tsconfigDefaults: {
       compilerOptions: {
-        sourceMap: true,
-        declaration: true
+        sourceMap: true
       },
     },
     tsconfigOverride: {
-      compilerOptions: {
+     exclude: [
```

Comment:
Not crazy on having typescript settings in multiple places. How about a tsconfig.json and tsconfig.build.json? It supports extending configs!

Reply:
These are literally just overrides. I'm not a big fan of adding multiple tsconfigs as that just adds another set of configs that has to be read when looking at the rollup config.

In this case this is just forcing declarations and excluding tests, which won't make much sense outside of rollup 🥳